### PR TITLE
Golang single service

### PR DESCRIPTION
**Pull request by sethjback**
This draft is a proposal for 2 things:

1. Fixing the package namespace issue in the protobuf. Ideally this will be pushed to a separate PR, it is included here simply to make everything work.
2. Creates a `trinsic-service` that wraps all the individual services

This is intended mostly to encourage feedback on the wrapper service

### CODE REVIEW

File: go/services/credential_service.go [hunk 1]
```diff
@@ -2,7 +2,8 @@ package services
 
 import (
 	"context"
-	credential "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/v1"
+
+	credentials "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/v1"
```
Comment (fundthmcalculus) [inline go/services/credential_service.go]: I specifically went through the languages and made all services singular, not plural. Is there a reason for this being reverted?

**Review by fundthmcalculus — APPROVED** [review]
In general, I like this. Except for the singular/plural issue, which might just be a personal preference.

File: go/services/credential_service.go (hunk 1 above)
Comment (sethjback) [reply go/services/credential_service.go]: Nope! Singular works - this is just my RESTful instinct coming out....

**Review by fundthmcalculus — CHANGES_REQUESTED** [review]
Blocking merge until @sethjback has time to confirm I didn't break anything

**Comment by sethjback** [comment]
Looks good to me

**Review by fundthmcalculus — APPROVED** [review]

**Pull request merged**